README.md

```diff
@@ -92,4 +92,4 @@ Free Software Foundation.
  3. Use British spelling, when appropriate.
  
  4. By contributing, you agree for your work to be included under
-    the terms of the program licence.
+    the terms of the program license.
```

Comment:
`licence` is the British spelling, `license` is the American one. We prefer British here.